packages/auth/package.json

```diff
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "@asap-hub/eslint-config-asap-hub": "workspace:packages/eslint-config-asap-hub",
-    "@auth0/auth0-spa-js": "^1.9.0"
+    "@auth0/auth0-spa-js": "^1.9.0",
```

Comment:
tbc back to 12 when rebasing on #211

Reply:
Can rebase on master now and use its something something 12.12 version everywhere :sweat_smile: